src/main/java/org/dcsa/bkg/repositories/BookingConfirmationRepository.java

```diff
@@ -0,0 +1,11 @@
+package org.dcsa.bkg.repositories;
+
+import org.dcsa.core.events.model.Shipment;
+import org.dcsa.core.repository.ExtendedRepository;
+import reactor.core.publisher.Mono;
+
+public interface BookingConfirmationRepository

```

Comment:
`BookingConfirmationRepository` is the same as `ShipmentRepository` which exists in event core, maybe use that instead.